test/Blazor.Extensions.Storage.Test/Blazor.Extensions.Storage.Test.csproj

```diff
@@ -1,15 +1,19 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <RunCommand>dotnet</RunCommand>
-    <RunArguments>blazor serve</RunArguments>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RestoreAdditionalProjectSources>
+      https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
```

Comment:
Why the dependency on MyGet?

Reply:
Sorry, copied it from the template.  Submitted a commit to fix.  Thanks!!